src/base/misc/fatfs.h

```diff
@@ -69,6 +71,14 @@ typedef struct {
 
 } fatfs_t;
 
+struct rxdos_loadstackvars {
```

Comment:
Please move this struct to where you use it.
It is not needed in a public header.

Reply:
Should it go near the top, or directly in front of the function that uses it?

Reply:
But with that option, how would I specify the cast to that pointer's type?

Reply:
Do you need to cast, LINEAR2UNIX() returns a void * ?

Also sizeof(struct rxdos_loadstackvars) could be sizeof(*lsv)